docker/fuzzbench/Dockerfile

```diff
@@ -31,7 +31,10 @@ COPY benchmarks benchmarks
 COPY common common
 COPY database database
 COPY docker docker
+COPY experiment/build experiment/build
+COPY experiment/*.py experiment/
```

Comment:
We are not going to copy all files in `experiment`.
Maybe in the future, these files should move out of `experiment`, where should be the old skeleton, independent with the new arch. And both archs base on same group of code.